src/main/java/Entity/statusEnum.java

```diff
@@ -0,0 +1,6 @@
+package Entity;
+
+public enum statusEnum {
```

Comment:
Usually good to have enums with a starting capital letter.

`StatusEnum`.

This allows us to differentiate easily what is a variable, and what is the enum.